test/cbmc/proofs/DNS/CreateDNSMessage/CreateDNSMessage_harness.c

```diff
@@ -0,0 +1,58 @@
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "queue.h"
+#include "list.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_DNS.h"
+#include "FreeRTOS_IP_Private.h"
+
+#include "cbmc.h"
+
+size_t prvCreateDNSMessage( uint8_t * pucUDPPayloadBuffer,
```

Comment:
Why is this re-declaration needed?

Reply:
prvCreateDNSMessage is a static function that is made non static just for CBMC proof with headers

Reply:
Yeah i think we have decided on exporting static function already as annirudh suggested.